puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/pr_checkout.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
+
+cd plugin/
```

Comment:
Tried a multi-purpose solution for plugin and non-plugin pr checkouts -- let me know what you think. If you would still prefer they be separated logic for plugins and non-plugins I'll generate two scripts.